Explorer/Assets/DCL/SDKComponents/Tween/Systems/TweenLoaderSystem.cs

```diff
@@ -49,7 +49,7 @@ private void UpdateTween(ref PBTween pbTween, ref SDKTweenComponent tweenCompone
             // Its present in the old renderer. If this was not needed, the CurrentTween field can be deleted
             if (pbTween.IsDirty || !TweenSDKComponentHelper.AreSameModels(pbTween, tweenComponent.CachedTween))
             {
-                tweenComponent.CachedTween = pbTween;
+                tweenComponent.CachedTween = new PBTween(pbTween);
```

Comment:
So if you want to make a copy you have `PBTween` should be poolable: I propose you to keep an instance in `tweenComponent` taken from the pool, then call `MergeFrom` instead of `ctor`, and return to the pool on component dispose